net/instaweb/rewriter/dedup_inlined_images_filter_test.cc

```diff
@@ -39,10 +39,10 @@ namespace net_instaweb {
 namespace {
 
 // Filenames of resource files.
-const char kCuppaPngFilename[] = "Cuppa.png";
-const char kPuzzleJpgFilename[] = "Puzzle.jpg";
+constexpr absl::string_view kCuppaPngFilename = "Cuppa.png";
```

Comment:
Are these changes necessary? In the original code this gets initialized by the compiler. It's not obvious to me what happens in the new system; I"m surprised it compiles actually. When does the string_view ctor get run?

Reply:
These are needed for analysis::StrFormat to get compile time checking of the format string. See https://abseil.io/docs/cpp/guides/format, which mentions it

Reply:
At least in this case, kCuppaPngFilename is not used as a format string; it's just used in StrCat.

If we do use a statically initialized string constant, would it work to declare that as
```
constexpr char kFoo[] = "foo";
```
?

Reply:
Done in 01f90d1a1b585f36f38d0d5c9da6525e28183525